src/actions/projectActions.js

```diff
@@ -40,29 +28,15 @@ export function syncUserProj(payload) {
 export const asyncExampleProj = () => {
     // fetch example projects
     return (dispatch) => {
-        let exampleVals = [];
-        scenes.where("uid", "==", "1").get().then(snap => {
-            let num_proj_loaded = 0;
-            snap.forEach(doc => {
-                storageRef.child(`/images/perspective/${doc.id}`)
-                    .getDownloadURL()
-                    .catch(() => {
-                        console.error("Error: Missing preview image");
-                    })
-                    .then((img) => {
-                        let dat = doc.data();
-                        exampleVals.push({
-                            name: dat.name,
-                            id: doc.id,
-                            data: dat,
-                            url: img ? img : "/img/no_preview.jpg"
-                        });
-                        num_proj_loaded++;
-                        if(num_proj_loaded === snap.size){
-                            dispatch(syncExampleProj(exampleVals));
-                        }
-                    });   
-            });
+        fetch(`${sceneRef}/`, {headers: {"x-access-token": "1"}}).then((response) =>{
```

Comment:
Again this is a backend comment on frontend code, but can we make an endpoint for examples that doesn't take a token? Such as `/apiv1/scenes/examples`